test/verify/check-shell-host-switching

```diff
@@ -117,8 +117,9 @@ class TestHostSwitching(testlib.MachineCase, HostSwitcherHelpers):
 
     def enable_multihost(self, machine):
         if not self.multihost_enabled:
-            machine.write("/etc/cockpit/shell.override.json",
-                          '{ "config": { "host_switcher": true } }')
+            machine.write("/etc/cockpit/cockpit.conf",
+                          '[WebService]\nAllowMultiHost=yes\n')
```

Comment:
Our machines setup regularly writes a cockpit.conf to disable TLS and such -- maybe that explains some of the test failures? (not sure). If so, perhaps `append=True` helps.